pkg/autoscaler/stats_reporter_test.go

```diff
@@ -73,58 +73,58 @@ func TestReporter_Report(t *testing.T) {
-	expectSuccess(t, func() error { return r.Report(DesiredPodCountM, 2) })
-	expectSuccess(t, func() error { return r.Report(DesiredPodCountM, 3) })
+	expectSuccess(t, "ReportDesiredPodCount", func() error { return r.ReportDesiredPodCount(1) })
+	expectSuccess(t, "ReportDesiredPodCount", func() error { return r.ReportDesiredPodCount(2) })
+	expectSuccess(t, "ReportDesiredPodCount", func() error { return r.ReportDesiredPodCount(3) })
 	checkData(t, "desired_pods", wantTags, 3)
 
-	expectSuccess(t, func() error { return r.Report(RequestedPodCountM, 4) })
-	expectSuccess(t, func() error { return r.Report(RequestedPodCountM, 5) })
-	expectSuccess(t, func() error { return r.Report(RequestedPodCountM, 6) })
+	expectSuccess(t, "ReportRequestedPodCount", func() error { return r.ReportRequestedPodCount(4) })
+	expectSuccess(t, "ReportRequestedPodCount", func() error { return r.ReportRequestedPodCount(5) })
+	expectSuccess(t, "ReportRequestedPodCount", func() error { return r.ReportRequestedPodCount(6) })
 	checkData(t, "requested_pods", wantTags, 6)
 
-	expectSuccess(t, func() error { return r.Report(ActualPodCountM, 7) })
-	expectSuccess(t, func() error { return r.Report(ActualPodCountM, 8) })
-	expectSuccess(t, func() error { return r.Report(ActualPodCountM, 9) })
+	expectSuccess(t, "ReportActualPodCount", func() error { return r.ReportActualPodCount(7) })
+	expectSuccess(t, "ReportActualPodCount", func() error { return r.ReportActualPodCount(8) })
+	expectSuccess(t, "ReportActualPodCount", func() error { return r.ReportActualPodCount(9) })
 	checkData(t, "actual_pods", wantTags, 9)
 
-	expectSuccess(t, func() error { return r.Report(PanicM, 1) })
-	expectSuccess(t, func() error { return r.Report(PanicM, 0) })
-	expectSuccess(t, func() error { return r.Report(PanicM, 1) })
+	expectSuccess(t, "ReportPanic", func() error { return r.ReportPanic(1) })
+	expectSuccess(t, "ReportPanic", func() error { return r.ReportPanic(0) })
+	expectSuccess(t, "ReportPanic", func() error { return r.ReportPanic(1) })
 	checkData(t, "panic_mode", wantTags, 1)
 
-	expectSuccess(t, func() error { return r.Report(PanicM, 0) })
+	expectSuccess(t, "ReportPanic", func() error { return r.ReportPanic(0) })
 	checkData(t, "panic_mode", wantTags, 0)
 }
 
-func expectSuccess(t *testing.T, f func() error) {
+func expectSuccess(t *testing.T, funcName string, f func() error) {
 	if err := f(); err != nil {
-		t.Errorf("Reporter.Report() expected success but got error %v", err)
+		t.Errorf("Reporter.%v() expected success but got error %v", funcName, err)
 	}
 }
 
 func checkData(t *testing.T, name string, wantTags map[string]string, wantValue float64) {
 	if d, err := view.RetrieveData(name); err != nil {
-		t.Errorf("Reporter.Report() error = %v, wantErr %v", err, false)
+		t.Errorf("Got unexpected error from view.RetrieveData error: %v", err)
 	} else {
 		if len(d) != 1 {
-			t.Errorf("Reporter.Report() len(d) %v, want %v", len(d), 1)
+			t.Errorf("want 1 data row but got %d", len(d))
```

Comment:
It is helpful to have additional context in the error as the the function. Also capitalize 'Want' for consistency with above. Same for the errors below.